lib/blocks.js

```diff
@@ -126,29 +189,55 @@ api.write = ({ledgerNode, state, consensusResult}, callback) => {
         delete block.eventHash;
 
         // TODO: ensure storage supports `consensusProof` event hash lookup
-
+        const now = Date.now();
         callback(null, {
           block: block,
-          meta: {blockHash}
+          meta: {blockHash, consensus: true, consensusDate: now}
         });
       });
     }],
-    store: ['createBlock', (results, callback) =>
+    updateKey: ['createBlock', (results, callback) => {
```

Comment:
Do we have a possible inconsistency issue with updating keys here as well?

If the block is written but we fail to update keys ... will that be a problem? I think the old "incomplete block" code used to prevent that somehow so I just want to double check that it's not an issue now with these changes. The repair cache code could also re-execute the update keys code as a possible fix (if needed).

Reply:
It is no longer possible for the block to be written without the keys being updated.  

https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity/blob/41e96f26fd6d6a30ebc1a81079901e5c8f3046de/lib/blocks.js#L189-L208

Reply:
If we fail after the keys are updated and later try to write the block again, will it have the same keys/key references in it as before?

Reply:
Good catch.  I added a provision to address this situation.